fuels-core/src/code_gen/functions_gen.rs

```diff
@@ -690,27 +630,6 @@ pub fn hello_world(
         assert_eq!(result, expected);
     }
 
-    #[test]
-    fn test_expand_function_argument_empty_fourth() {
```

Comment:
why is this not needed anymore?

Reply:
https://github.com/FuelLabs/fuels-rs/pull/150/files#r825151275